modules/hosted-control-planes-troubleshooting.adoc

```diff
@@ -40,57 +40,13 @@ Although the output does not contain any secret objects from the cluster, it can
-    --name <hosted_cluster_name> \// <1>
-    --namespace <hosted_cluster_namespace> \ <2>
-    --dump-guest-cluster \
-    --artifact-dir clusterDump-<hosted_cluster_namespace>-<hosted_cluster_name>
+$ *oc adm must-gather --image=registry.redhat.io/multicluster-engine/must-gather-rhel8:v__<mce_version>__ /usr/bin/gather hosted-cluster-namespace=HOSTEDCLUSTERNAMESPACE hosted-cluster-name=HOSTEDCLUSTERNAME --dest-dir=NAME ; tar -cvzf NAME.tgz NAME*
 ----
 +
-<1> Specify your hosted cluster name.
-<2> Specify your hosted cluster namespace, for example, `clusters`.
-+
-.Example output
-+
-[source,terminal]
-----
-2023-06-06T12:18:20+02:00   INFO    Archiving dump  {"command": "tar", "args": ["-cvzf", "hypershift-dump.tar.gz", "cluster-scoped-resources", "event-filter.html", "namespaces", "network_logs", "timestamp"]}
-2023-06-06T12:18:21+02:00   INFO    Successfully archived dump  {"duration": "1.519376292s"}
-----
+where:
 
-* To configure the command-line interface so that it impersonates all of the queries against the management cluster by using a username or service account, enter the `hypershift dump cluster` command with the `--as` flag.
-+
-The service account must have enough permissions to query all of the objects from the namespaces, so the `cluster-admin` role is recommended to make sure you have enough permissions. The service account must be located in or have permissions to query the namespace of the `HostedControlPlane` resource.
-+
-If your username or service account does not have enough permissions, the output contains only the objects that you have permissions to access. During that process, you might see `forbidden` errors.
-+
-** To use impersonation by using a service account, enter the following command:
-+
-[source,terminal]
-----
-$ hypershift dump cluster \
-    --name <hosted_cluster_name> \// <1>
-    --namespace <hosted_cluster_namespace> \// <2>
-    --dump-guest-cluster \
-    --as "system:serviceaccount:<service_account_namespace>:<service_account_name>" \ <3>
-    --artifact-dir clusterDump-<hosted_cluster_namespace>-<hosted_cluster_name>
-----
-<1> Specify your hosted cluster name.
-<2> Specify your hosted cluster namespace, for example, `clusters`.
-<3> Specify the `default` namespace and name, for example, `"system:serviceaccount:default:samplesa"`.
-
-** To use impersonation by using a username, enter the following command:
-+
-[source,terminal]
-----
-$ hypershift dump cluster \
-    --name <hosted_cluster_name> \// <1>
-    --namespace <hosted_cluster_namespace> \// <2>
-    --dump-guest-cluster \
-    --as "<cluster_user_name>" \ <3>
-    --artifact-dir clusterDump-<hosted_cluster_namespace>-<hosted_cluster_name>
-----
-<1> Specify your hosted cluster name.
-<2> Specify your hosted cluster namespace, for example, `clusters`.
-<3> Specify your cluster user name, for example, `cloud-admin`.
+** You replace `<mce_version>` with the version of {mce-short} that you are using; for example, `v2.6`.
```

Comment:
nit: should 'v' be removed here or maybe up top? Just trying to avoid someone typing vv2.6 😆

Reply:
Good point! I removed it from line 50.